tests/baselines/reference/api/typescript.d.ts

```diff
@@ -7912,9 +7912,27 @@ declare namespace ts {
 }
 declare namespace ts {
     interface Node {
-        /** @deprecated `decorators` has been merged with `modifiers` on the declarations that support decorators. */
-        readonly decorators?: NodeArray<Decorator> | undefined;
-        /** @deprecated `modifiers` has been removed from `Node` and moved to the specific `Node` subtypes that support them. */
+        /**
+         * @deprecated `decorators` has been merged with `modifiers` on the declarations that support decorators.
+         * To test whether a `Node` can legally have decorators, use the `ts.canHaveDecorators()` function. To read the
+         * decorators of a `Node` that can legally have decorators, use the `ts.getDecorators()` function.
+         *
+         * For example:
+         * ```ts
+         * const decorators = ts.canHaveDecorators(node) ? ts.getDecorators(node) : undefined;
+         * ```
+         */
+        readonly decorators: never;
+        /**
+         * @deprecated `modifiers` has been removed from `Node` and moved to the specific `Node` subtypes that support them.
```

Comment:
are `modifiers` actually removed? or just more specific in their type on sub-nodes?

Reply:
It's intended that `modifiers` is not a property of every `Node`, but rather is a property of specific `Node` sub-types. i.e., you shouldn't just blindly read from `node.modifiers`.